beacon-chain/db/kv/blocks.go

```diff
@@ -266,7 +265,7 @@ func (k *Store) SaveBlock(ctx context.Context, block *ethpb.BeaconBlock) error {
 		if err := updateValueForIndices(indicesByBucket, blockRoot[:], tx); err != nil {
 			return errors.Wrap(err, "could not update DB indices")
 		}
-		k.blockCache.Set(string(blockRoot[:]), block, time.Hour)
+		k.blockCache.Set(string(blockRoot[:]), block, 0)
```

Comment:
Why is cost zero?

```
// To dynamically evaluate the items cost using the Config.Coster function, set
// the cost parameter to 0 and Coster will be ran when needed in order to find
// the items true cost.
```

Isn't that going to be slower?

Reply:
Actually, it looks like it won't run any function if there isn't one defined, but it will leave cost at zero. So the cache would never start eviction since all items are zero cache?